docs/README.md

```diff
@@ -46,6 +57,14 @@ Install ty into your current Python environment with pip:
 pip install ty
 ```
 
+### Running ty
+
+Run [`ty check`](./reference/cli.md#ty-check), in your project's top-level directory,
+to check the project for type errors using ty's default configuration.
+
+If this provokes a cascade of errors, and you are using `venv` for a virtual environment, add
```

Comment:
maybe "the standard-libary `venv` module"?

Reply:
Thanks. Fixed.

Reply:
thank you!